firebase_admin/_user_identifier.py

```diff
@@ -30,7 +32,12 @@ def __init__(self, uid):
         Args:
             uid: A user ID string.
         """
-        self.uid = uid
+        _auth_utils.validate_uid(uid, required=True)
+        self._uid = uid
```

Comment:
You can collapse these into a single line:

```
self._uid = _auth_utils.validate_uid(...)
```

Same below.

Reply:
Done.